benchmarks/TFLite/CMakeLists.txt

```diff
@@ -132,6 +132,32 @@ set(MOBILENET_V2_MODULE
     "1x224x224x3xf32"
 )
 
+set(MOBILENET_V2_UINT8_MODULE
+  NAME
+    "MobileNetV2Uint8"
+  TAGS
+    "uint8,imagenet"
+  SOURCE
+    "https://storage.googleapis.com/iree-model-artifacts/mobilenet_v2_224_1.0_uint8.tflite"
+  ENTRY_FUNCTION
+    "main"
+  FUNCTION_INPUTS
+    "1x224x224x3xui8"
+)
+
+set(MOBILENET_V2_INT8_MODULE
+  NAME
+    "MobileNetV2Int8"
+  TAGS
+    "int8,imagenet"
+  SOURCE
+    "https://storage.googleapis.com/tf_model_garden/vision/mobilenet/v2_1.0_int8/mobilenet_v2_1.00_224_int8.tflite"
+  ENTRY_FUNCTION
+    "main"
+  FUNCTION_INPUTS
+    "1x224x224x3xf32"
```

Comment:
Does this INT8 model take in F32 input?

Reply:
Surprisingly yes:

```mlir
func.func @main(%arg0: tensor<?x224x224x3xf32> {iree.identifier = "serving_default_input_9:0", tf_saved_model.index_path = ["input_9"]}) -> (tensor<?x1001xf32> {iree.identifier = "StatefulPartitionedCall:0", tf_saved_model.index_path = ["quant_dense_5"]}) attributes {tf_saved_model.exported_names = ["serving_default"]} {
```

At 3 op it gets converted to i8 and then no f32 is used until the 3rd to last it gets converted back to float, so all operations is integral but inputs & output in float.